pkg/fees/txFees.test.ts

```diff
@@ -0,0 +1,51 @@
+import { getFeesBTC } from "./txFees"
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+jest.mock("node-fetch", () => require("fetch-mock-jest").sandbox())
```

Comment:
We'll be using axios going forward and mock this in the new Blockstream class.
Nico and I will finish that hopefully tomorrow.